EST.MIT.Payment.Models/PaymentRequest.cs

```diff
@@ -6,25 +6,39 @@ public class PaymentRequest
     [JsonProperty("paymentRequestId")]
     public string PaymentRequestId { get; init; } = default!;
     [JsonProperty("frn")]
-    public int FRN { get; init; }
+    public long FRN { get; init; }
+    [JsonProperty("sbi")]
+    public int SBI { get; init; }
+    [JsonProperty("vendor")]
+    public string Vendor { get; init; } = default!;
     [JsonProperty("sourceSystem")]
     public string SourceSystem { get; init; } = default!;
     [JsonProperty("marketingYear")]
     public int MarketingYear { get; init; }
-    [JsonProperty("deliveryBody")]
-    public string DeliveryBody { get; init; } = default!;
+    [JsonProperty("currency")]
+    public string Currency { get; init; } = default!;   
+    [JsonProperty("description")]
+    public string Description { get; init; } = default!;
+    [JsonProperty("originalInvoiceNumber")]
+    public string OriginalInvoiceNumber { get; init; } = default!;  
+    [JsonProperty("originalSettlementDate")]
+    public string OriginalSettlementDate { get; init; } = default!; 
```

Comment:
OriginalSettlementDate amd RecoveryDate should be DateTime. (I thought RecoveryDate was DateTime just now?)

Reply:
resolved